public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'formik';
+import SubHeader from '../../../../../components/SubHeader';
+import { FormikSelect } from '../../../../../components/FormControls';
+
+
+const methodOptions = [{ value: 'POST', text: 'POST' }, { value: 'PUT', text: 'PUT' }, { value: 'PATCH', text: 'PATCH' }, { value: 'GET', text: 'GET' }, { value: 'DELETE', text: 'DELETE' }];
```

Comment:
Remove `DELETE` and `GET`, as we deleted those from the backend changes

Reply:
Thanks! Removed.